services/todos.js

```diff
@@ -1,44 +1,55 @@
 'use strict'
 
-const {BadRequestError, DBProblemError} = require('./errors');
-const prismaSingleton = require('./prisma');
-const prisma = prismaSingleton();
+const users_services = require('./users');
+const {BadRequestError} = require('./errors');
+const pgObject = require('./pg');
 
 const validateTodo = function(todo){
-    if ((typeof todo.user_id === 'number')&&(typeof todo.name === 'string')){
+    if ((typeof todo.user_email === 'string')&&(typeof todo.name === 'string')){
         return todo;
     }else{
         throw new BadRequestError("Minimum user properties is missing");
     }
 }
```

Comment:
Sugestão opcional:

O `else` aqui é desnecessário!
```suggestion
    if ((typeof todo.user_email === 'string')&&(typeof todo.name === 'string')){
        return todo;
    }
    
    throw new BadRequestError("Minimum user properties is missing");
}
```